<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="image">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Select an image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>File selected:none</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>203</width>
     <height>96</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Label">
        <property name="text">
         <string>No of white pixels   :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noOfWhitePixels">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Label_2">
        <property name="text">
         <string>No of black pixels    :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noOfBlackPixels">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Label_3">
        <property name="text">
         <string>No of other pixels    :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="noOfOtherPixels">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="Label_4">
        <property name="text">
         <string>Total pixels                 :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalPixels">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
